code/go/0chain.net/blobber/zcn.go

```diff
@@ -36,6 +37,14 @@ func registerOnChain() error {
 		return err
 	}
 
+	_, err = sdk.GetBlobber(node.Self.ID)
```

Comment:
It returns `"error": "sdk_not_initialized: SDK is not initialised"` error, which makes validators try to register every time on restarting